packages/webapp/src/members/components/members-grid.tsx

```diff
@@ -21,8 +21,8 @@ export const MembersGrid = ({ members, dataTestId }: Props) => {
     return (
         <div className={containerClass} data-testid={dataTestId}>
             {(members.length &&
-                members.map((member, index) => (
-                    <MemberSquare key={index} member={member} />
+                members.map((member) => (
+                    <MemberSquare key={member.account} member={member} />
```

Comment:
This one is kinda dangerous because we might reuse this MembersGrid to show cards (with different NFTs#) to show the sales/auctions for a single member... I'm fine with merging it now, but we just need to be careful to not reuse the grid in case same member cards is repeated.